packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts

```diff
@@ -148,22 +151,23 @@ export function getStyles(props: IRatingStyleProps): IRatingStyles {
       classNames.ratingStarIsSmall,
       {
         fontSize: ratingSmallIconSize + 'px',
-        lineHeight: ratingSmallIconSize + 'px'
+        lineHeight: ratingSmallIconSize + 'px',
+        height: ratingSmallIconSize + 'px'
       }
     ],
     ratingStarIsLarge: [
       classNames.ratingStartIsLarge,
       {
         fontSize: ratingLargeIconSize + 'px',
-        lineHeight: ratingLargeIconSize + 'px'
+        lineHeight: ratingLargeIconSize + 'px',
+        height: ratingLargeIconSize + 'px'
       }
     ],
     labelText: [classNames.labelText, hiddenContentStyle],
     ratingFocusZone: [
       classNames.ratingFocusZone,
       {
-        display: 'inline-block',
-        paddingBottom: '1px'
```

Comment:
Removed this 1px padding as it was interfering with the overall height of the Rating wrapper. Have no idea why it even was needed in here.